airflow/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -38,7 +38,8 @@
 OUTPUT_PATH = "gs://{}/{}/".format(BUCKET, OUTPUT_FOLDER)
 PYSPARK_MAIN = os.environ.get("PYSPARK_MAIN", "hello_world.py")
 PYSPARK_URI = "gs://{}/{}".format(BUCKET, PYSPARK_MAIN)
-
+SPARKR_MAIN = os.environ.get("SPARKR_MAIN", "hello_world.R")
```

Comment:
I'm not actually sure what environ var I should use here? SPARKR_RLIBDIR ?

Reply:
or SPARKR_DRIVER_R?

Reply:
I'm ok with `SPARKR_MAIN`

Reply:
I will have to check what environmental var is set in dataproc. because this var refers to the main location of the executable right?

Reply:
The environment variable is taken from the environment where you run Airflow, not from Dataproc.